image_classification/mobilenet_nchw.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import {buildConstantByNpy} from '../common/utils.js';
+
+// MobileNet V2 baseline model with nchw layout
+export class MobileNetNchw {
+  constructor() {
+    this.builder_ = null;
+    this.graph_ = null;
+  }
+
+  async buildConv_(input, name, shouldClip = true, options = undefined) {
+    const prefix = './weights/mobilenet_nchw/conv_' + name;
+    const weightsName = prefix + '_weight.npy';
+    const weights =
+        await buildConstantByNpy(this.builder_, weightsName);
+    const biasName = prefix + '_bias.npy';
+    const bias =
+        await buildConstantByNpy(this.builder_, biasName);
+    const conv = this.builder_.add(
+        this.builder_.conv2d(input, weights, options),
+        this.builder_.reshape(bias, [1, -1, 1, 1]));
+    if (shouldClip) {
+      return this.builder_.clamp(
+          conv,
+          {
+            minValue: this.builder_.constant(0.),
+            maxValue: this.builder_.constant(6.0),
+          });
+    } else {
+      return conv;
+    }
+  }
+
+  async buildGemm_(input, name) {
+    const prefix = './weights/mobilenet_nchw/gemm_' + name;
+    const weightsName = prefix + '_weight.npy';
+    const weights = await buildConstantByNpy(this.builder_, weightsName);
+    const biasName = prefix + '_bias.npy';
+    const bias = await buildConstantByNpy(this.builder_, biasName);
+    const options = {c: bias, bTranspose: true};
+    return this.builder_.gemm(input, weights, options);
+  }
+
+  async buildFire_(
```

Comment:
`Fire` is not used in MobileNetV2 (it's used for SqueezeNet). According to the [paper](https://arxiv.org/pdf/1801.04381.pdf), we should build `Bottleneck` block.  And there are two types of bottleneck block, as it shows in the diagram:

![image](https://user-images.githubusercontent.com/1005673/114963686-df672680-9e9f-11eb-8958-42e5af966440.png)

And I'd suggest to create a helper to build depthwise conv explicitly. The helper could create grouped conv with groups == input channels.